process-engine/skeleton
# :arrow_up: update dependencies

## What did you change?

I updated the dependency-versions to use the latest process-engine-packages

## How can others test the changes?

Use the regular skeleton-setup, and see how it no works with versioning and namespaces

## PR-Checklist

Please check the boxes in this list after submitting your PR:
